Pe bug fixes (#253)
AY2021S1-CS2103T-T10-4/tp: docs/UserGuide.md

  ![Add lesson after](images/ugimages/AddLessonAfter.png)

<div markdown="block" class="alert alert-info">

**:information_source: Notes about the command:**<br>
  * You must type out the day in full.
  * You are not allowed to type in overnight lessons. For example, a lesson cannot start at `23:59` and end at `00:01`.
  * The `Number of Weeks` displayed refers to the total number of weeks you will be teaching the lesson.
    It does not follow the week numbers of a semester.

>     For example, if a semester has 13 weeks and your lessons begin from week 3 and end at week 13,

Reviewer: Perhaps `in` would make it more obvious if it is inclusive or not.
```suggestion
    For example, if a semester has 13 weeks and your lessons begin in week 3 and end in week 13,
```
